### PR TITLE
Split pulleys

### PR DESCRIPTION
**Pull request by matthew-reynolds**
- Expanded pulley width from 20mm to 22mm
 - Shrunk pulley walls from 2mm to 1mm
 - Made pulleys print in 2 parts to avoid overhangs

image
image

image
image

### CODE REVIEW

**Comment by matthew-reynolds** [comment]
smh I forgot to un-suppress a couple mates in the leg assy, but #67 depends on this PR, so I'll just wait until they're merged and then put up another PR lol 10/10 development workflow

**Review by dawsonhemphill — APPROVED** [review]

**Comment by wmmc88** [comment]
reasoning behind moving to 1mm? I guess there should be no force there so should be fine anyways.

**Review by wmmc88 — APPROVED** [review]

**Comment by matthew-reynolds** [comment]
> reasoning behind moving to 1mm? I guess there should be no force there so should be fine anyways.

Just to keep the overall width unchanged at 24mm, otherwise I'd need to extend the knee motor bracket by a couple mm to keep the two pulleys in line. (not a big deal, but would rather not if not necessary)

**Pull request merged**
